README.md

```diff
@@ -440,6 +495,69 @@ Many decisions around granting equity to employees are approved by the board of
 
 🚧 This section could be expanded, and also include more legal links.
 
+### IPOs
```

Comment:
Public and private companies followed by how private companies might become public companies 👍